### PR TITLE
chore(Typography): move to own subfolder

### PR DESCRIPTION
**Pull request by joakbjerk**

### CODE REVIEW

File: packages/dnb-eufemia/src/elements/typography/P.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  | 'basis'
+  | 'medium'
+  | 'large'
+  | 'x-large'
+  | 'xx-large'
+
+export type PProps = SpacingProps &
+  React.HTMLAttributes<HTMLParagraphElement> & {
+    element?: DynamicElement & 'p'
+    small?: boolean
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/elements/typography/P.tsx:24]: I think we should add docs (comments) each prop.

```suggestion
    /**
     * Formatted like other TS components (e.g. breadcrumb)
     */
    small?: boolean
```

**Review by tujoworker — APPROVED** [review]
Well done 👏 (just some minor wishes from my side). You can add these also i a new PR if you want 👍 )

File: packages/dnb-eufemia/src/elements/typography/H.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  | 'medium'
+  | 'basis'
+  | 'small'
+  | 'x-small'
+
+type HProps = SpacingProps &
+  React.HTMLAttributes<HTMLHeadingElement> & {
+    as?: string
+    level?: string
+    size?: HSize
```
[the remark this answers is not in the document]
Comment (joakbjerk) [reply packages/dnb-eufemia/src/elements/typography/H.tsx:25]: Same here, not sure if I have the required knowledge for proper documentation

File: packages/dnb-eufemia/src/elements/typography/P.tsx (hunk 1 above)
Comment (joakbjerk) [reply packages/dnb-eufemia/src/elements/typography/P.tsx:24]: Sounds good, but I'm not sure if I understand the props well enough to properly document them 😅

Comment (tujoworker) [reply packages/dnb-eufemia/src/elements/typography/P.tsx:24]: Lets do it in follow up PR. Try first by yourself with an PR. We will then double check for sure. You can also check existing documentation. Like for the level, we have the other Heading component, witch also supports `level`.

File: packages/dnb-eufemia/src/elements/Paragraph.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/**
+ * This file is created manually
+ * You can change the content
+ *
+ */
+
+import Paragraph from './typography/Paragraph'
+export * from './typography/Paragraph'
+export default Paragraph
```
[the remark this answers is not in the document]
Comment (joakbjerk) [reply packages/dnb-eufemia/src/elements/Paragraph.ts:9]: Should removing this element be its own PR too? or should I do it in this one, if we have come to a conclusion yet? 😄

[the remark this answers is not in the document]
Comment (langz) [reply packages/dnb-eufemia/src/elements/Paragraph.ts:9]: I don't really have any opinion regarding this for now.
I do however see that it exists quite a few references to `import { Paragraph } from '@dnb/eufemia/elements'` in the monorepo dnb-web 🤔

[the remark this answers is not in the document]
Comment (joakbjerk) [reply packages/dnb-eufemia/src/elements/Paragraph.ts:9]: Then I wote for eventually removing this one in a different PR 😄

**Pull request merged**